# disallow `Array.from` method (es/no-array-from)

This rule reports ES2015 `Array.from` method as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const array = Array.from("hello")
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-array-from.js)
- [Test source](../../tests/lib/rules/no-array-from.js)
