# disallow the `Atomics` class (es/no-atomics)

This rule reports ES2017 `Atomics` class as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
Atomics.add(buffer, 0, 2)
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-atomics.js)
- [Test source](../../tests/lib/rules/no-atomics.js)
