# disallow block-scoped variable declarations (es/no-block-scoped-variables)

This rule reports ES2015 block-scoped variable declarations as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
let a = 1
const b = 2
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-block-scoped-variables.js)
- [Test source](../../tests/lib/rules/no-block-scoped-variables.js)
