# disallow binary numeric literals (es/no-binary-numeric-literals)

This rule reports ES2015 binary numeric literals as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
let a = 0b1010
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-binary-numeric-literals.js)
- [Test source](../../tests/lib/rules/no-binary-numeric-literals.js)
