# disallow class declarations (es/no-classes)

This rule reports ES2015 class declarations as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
class A {}
const B = class {}
```
