# disallow default parameters (es/no-default-parameters)

This rule reports ES2015 default parameters as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
function f(a = 1) {}
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-default-parameters.js)
- [Test source](../../tests/lib/rules/no-default-parameters.js)
