# disallow default parameters (es/no-default-parameters)

This rule reports ES2015 default parameters as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
function f(a = 1) {}
```
