# disallow exponential operators (es/no-exponential-operators)

This rule reports ES2016 [exponential operators](https://github.com/rwaldron/exponentiation-operator#readme) as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
let a = a ** 2
let a **= b
```
