# disallow the `Map` class (es/no-map)

This rule reports ES2015 `Map` class as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
let map = new Map()
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-map.js)
- [Test source](../../tests/lib/rules/no-map.js)
