# disallow the `Math.hypot` method (es/no-math-hypot)

This rule reports ES2015 `Math.hypot` method as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const n = Math.hypot(value)
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-math-hypot.js)
- [Test source](../../tests/lib/rules/no-math-hypot.js)
