# disallow `Math.cosh` method (es/no-math-cosh)

This rule reports ES2015 `Math.cosh` method as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const n = Math.cosh(value)
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-math-cosh.js)
- [Test source](../../tests/lib/rules/no-math-cosh.js)
