# disallow the `Math.trunc` method (es/no-math-trunc)

This rule reports ES2015 `Math.trunc` method as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const n = Math.trunc(value)
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-math-trunc.js)
- [Test source](../../tests/lib/rules/no-math-trunc.js)
