# disallow `Math.log2` method (es/no-math-log2)

This rule reports ES2015 `Math.log2` method as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const n = Math.log2(value)
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-math-log2.js)
- [Test source](../../tests/lib/rules/no-math-log2.js)
