# disallow `Math.imul` method (es/no-math-imul)

This rule reports ES2015 `Math.imul` method as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const n = Math.imul(value)
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-math-imul.js)
- [Test source](../../tests/lib/rules/no-math-imul.js)
