# disallow the `Math.acosh` method (es/no-math-acosh)

This rule reports ES2015 `Math.acosh` method as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const n = Math.acosh(value)
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-math-acosh.js)
- [Test source](../../tests/lib/rules/no-math-acosh.js)
