# disallow the `Math.cbrt` method (es/no-math-cbrt)

This rule reports ES2015 `Math.cbrt` method as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const n = Math.cbrt(value)
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-math-cbrt.js)
- [Test source](../../tests/lib/rules/no-math-cbrt.js)
