# disallow `Math.atanh` method (es/no-math-atanh)

This rule reports ES2015 `Math.atanh` method as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const n = Math.atanh(value)
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-math-atanh.js)
- [Test source](../../tests/lib/rules/no-math-atanh.js)
