# disallow `Math.asinh` method (es/no-math-asinh)

This rule reports ES2015 `Math.asinh` method as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const n = Math.asinh(value)
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-math-asinh.js)
- [Test source](../../tests/lib/rules/no-math-asinh.js)
