# disallow the `Math.log10` method (es/no-math-log10)

This rule reports ES2015 `Math.log10` method as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const n = Math.log10(value)
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-math-log10.js)
- [Test source](../../tests/lib/rules/no-math-log10.js)
