# disallow `Math.sign` method (es/no-math-sign)

This rule reports ES2015 `Math.sign` method as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const n = Math.sign(value)
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-math-sign.js)
- [Test source](../../tests/lib/rules/no-math-sign.js)
