# disallow `Math.sinh` method (es/no-math-sinh)

This rule reports ES2015 `Math.sinh` method as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const n = Math.sinh(value)
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-math-sinh.js)
- [Test source](../../tests/lib/rules/no-math-sinh.js)
