# disallow `Number.isNaN` method (es/no-number-isnan)

This rule reports ES2015 `Number.isNaN` method as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const b = Number.isNaN(value)
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-number-isnan.js)
- [Test source](../../tests/lib/rules/no-number-isnan.js)
