# disallow the `Number.MAX_SAFE_INTEGER` property (es/no-number-maxsafeinteger)

This rule reports ES2015 `Number.MAX_SAFE_INTEGER` property as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const b = Number.MAX_SAFE_INTEGER
```

## ðŸ“š References

- [Rule source](https://github.com/mysticatea/eslint-plugin-es/blob/v1.2.0/lib/rules/no-number-maxsafeinteger.js)
- [Test source](https://github.com/mysticatea/eslint-plugin-es/blob/v1.2.0/tests/lib/rules/no-number-maxsafeinteger.js)
