# disallow `Number.MIN_SAFE_INTEGER` property (es/no-number-minsafeinteger)

This rule reports ES2015 `Number.MIN_SAFE_INTEGER` property as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const b = Number.MIN_SAFE_INTEGER
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-number-minsafeinteger.js)
- [Test source](../../tests/lib/rules/no-number-minsafeinteger.js)
