# disallow the `Number.isSafeInteger` method (es/no-number-issafeinteger)

This rule reports ES2015 `Number.isSafeInteger` method as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const b = Number.isSafeInteger(value)
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-number-issafeinteger.js)
- [Test source](../../tests/lib/rules/no-number-issafeinteger.js)
