# disallow `Number.EPSILON` property (es/no-number-epsilon)

This rule reports ES2015 `Number.EPSILON` property as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const b = Number.EPSILON
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-number-epsilon.js)
- [Test source](../../tests/lib/rules/no-number-epsilon.js)
