# disallow the `Number.parseInt` method (es/no-number-parseint)

This rule reports ES2015 `Number.parseInt` method as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const b = Number.parseInt(value)
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-number-parseint.js)
- [Test source](../../tests/lib/rules/no-number-parseint.js)
