# disallow octal numeric literals (es/no-octal-numeric-literals)

This rule reports ES2015 octal numeric literals as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
let a = 0o123
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-octal-numeric-literals.js)
- [Test source](../../tests/lib/rules/no-octal-numeric-literals.js)
