# disallow the `Object.setPrototypeOf` method (es/no-object-setprototypeof)

This rule reports ES2015 `Object.setPrototypeOf` as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
Object.setPrototypeOf(obj, proto)
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-object-setprototypeof.js)
- [Test source](../../tests/lib/rules/no-object-setprototypeof.js)
