# disallow the `Promise` class (es/no-promise)

This rule reports ES2015 `Promise` class as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
let p = new Promise()
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-promise.js)
- [Test source](../../tests/lib/rules/no-promise.js)
