# disallow RegExp named capture groups (es/no-regexp-named-capture-groups)

This rule reports ES2018 [RegExp named capture groups](https://github.com/tc39/proposal-regexp-named-groups#readme) as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const r1 = /(?<a>b)c/
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-regexp-named-capture-groups.js)
- [Test source](../../tests/lib/rules/no-regexp-named-capture-groups.js)
