# disallow the `Set` class (es/no-set)

This rule reports ES2015 `Set` class as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
let set = new Set()
```

## ðŸ“š References

- [Rule source](../../lib/rules/no-set.js)
- [Test source](../../tests/lib/rules/no-set.js)
