# disallow the `Set` class (es/no-set)

This rule reports ES2015 `Set` class as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
let set = new Set()
```

## ðŸ“š References

- [Rule source](https://github.com/mysticatea/eslint-plugin-es/blob/v1.2.0/lib/rules/no-set.js)
- [Test source](https://github.com/mysticatea/eslint-plugin-es/blob/v1.2.0/tests/lib/rules/no-set.js)
