# disallow template literals (es/no-template-literals)

This rule reports ES2015 template literals as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
const a1 = `foo`
const a2 = tag`foo`
```
